src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -112,10 +112,10 @@ void UpdateHVACInterface(EnergyPlusData &state,
     if ((CalledFrom == DataConvergParams::CalledFrom::AirSystemDemandSide) && (OutletNode == 0)) {
         // Air loop has no return path - only check mass flow and then set return inlet node mass flow to sum of demand side inlet nodes
 
-        airLoopConv.HVACMassFlowNotConverged[0] = false;
-        airLoopConv.HVACHumRatNotConverged[0] = false;
-        airLoopConv.HVACTempNotConverged[0] = false;
-        airLoopConv.HVACEnergyNotConverged[0] = false;
+        airLoopConv.HVACMassFlowNotConverged[(int)CalledFrom] = false;
```

Comment:
Doesn't need to be changed here, but you are welcome to do this cast one time and create a new int from it.  That way the array lookup is a bit cleaner.  It won't affect performance or anything, but could help readability:

```c++
    int iCall = (int)calledFromEnum;
    airLoopConv.HVACMassFlowNotConverged[iCall] = false;
    // ...lots more uses of iCall without having (int) everywhere...
```